shared/components/SiteFooter/index.scss

```diff
@@ -9,6 +9,7 @@
 .SiteFooter {
   border-top: 3px solid $grey;
   display: flex;
+  width: 100%;
 
   > * {
     width: 100%;
```

Comment:
could you apply this width 100% to the two elements directly, and see if line 12 is needed?
I'm worried about this selector `> * {` causing problems in the future.